packs/feats/you-dont-smell-right.json

```diff
@@ -29,6 +29,26 @@
             "title": "Pathfinder Howl of the Wild"
         },
         "rules": [
+            {
+                "adjustName": false,
+                "choices": [
+                    {
+                        "label": "PF2E.SpecificRule.Werecreature.FeralSenses.ChoiceSet.Grant",
+                        "predicate": [
+                            {
+                                "not": "feat:feral-senses"
+                            }
+                        ],
+                        "value": 30
+                    },
+                    {
+                        "label": "PF2E.SpecificRule.Werecreature.FeralSenses.ChoiceSet.Extend",
+                        "value": 60
+                    }
+                ],
+                "flag": "range",
+                "key": "ChoiceSet"
+            },
```

Comment:
Not sold on this since there's more ways than this archetype to get a scent sense, and if we do go with it both should be predicated to skip the choice.

Reply:
I just kept it because it was already automatic before. Unlike before, however, there is now at least an option to choose if you have scent from another source. But yes, I can remove the automation completely. Your decision.

I wouldn't see it as a final solution anyway, because it is always assumed that you previously had a scent range of 30 feet. But as far as I know, you can't currently extend the range of a sense using RE, you can only set an absolute value. But this should cover most cases at least now.